### ISSUE TITLE
Allow users to choose their build tool

### ISSUE THREAD

**Issue by swirlycheetah** [swirlycheetah/generator-angular2#4]
Expand upon single option to allow users to choose their build tool. 

Allow choices

* Gulp 
* Grunt

**Comment by ocombe** [comment]
Webpack would be cool too :)

**Comment by swirlycheetah** [comment]
Yeah it would, I've got an sample Webpack repository (not published yet) working with ng2 so integration shouldn't be too difficult.

I'm waiting for things to stabilise a bit more in terms of how ng2 is built & packaged before tackling this one.

**Comment by ocombe** [comment]
Yeah, I wanted to do my own skeleton app, but I'm waiting on the new router for now :)
